packages/payment-detection/package.json

```diff
@@ -57,7 +57,7 @@
     "@graphql-codegen/cli": "4.0.1",
     "@graphql-codegen/typescript": "4.0.1",
     "@graphql-codegen/typescript-document-nodes": "4.0.1",
-    "@graphql-codegen/typescript-graphql-request": "6.0.1",
+    "@graphql-codegen/typescript-graphql-request": "6.2.0",
```

Comment:
Fixes the generated GraphQL code that previously contained `imports` not compatible with latest module resolution.